treeherder/push_health/utils.py

```diff
@@ -0,0 +1,60 @@
+def clean_test(test_name):
```

Comment:
I shamelessly stole this code from @jmaher in https://github.com/jmaher/web-extensions/blob/master/greener_pastures/analyze_commit.py#L209

Reply:
no worries- I assume we can optimize this- I have been analyzing python code in the last 2 weeks for another project and have been learning some tricks- lets get this working first, then we can make it work at scale